public/susi-chatbot.js

```diff
@@ -35,125 +35,132 @@ let botbuilderBotMessageTextColor = '#455a64';
+      ? settings.botIconImage
+      : botbuilderIconImg;
+    applyTheme();
+  }
+}
+
 function getTheme() {
-  $.ajax({
-    type: 'GET',
-    url: `${api_url}/cms/getSkillMetadata.json?userid=${userid}&group=${group}&language=${language}&skill=${skill}`,
-    jsonpCallback: 'pa',
-    contentType: 'application/json',
-    dataType: 'jsonp',
-    jsonp: 'callback',
-    crossDomain: true,
-    success: function(data) {
-      if (!data.accepted) {
-        // not allowed to use. Exit.
-        console.log(data.message);
-        return;
-      }
-      enableBot();
-      if (data.skill_metadata && data.skill_metadata.design) {
-        let settings = data.skill_metadata.design;
-        botbuilderBackgroundBody = settings.bodyBackground
-          ? settings.bodyBackground
-          : botbuilderBackgroundBody;
-        botbuilderBodyBackgroundImg = settings.bodyBackgroundImage
-          ? settings.bodyBackgroundImage
-          : botbuilderBodyBackgroundImg;
-        botbuilderUserMessageBackground = settings.userMessageBoxBackground
-          ? settings.userMessageBoxBackground
-          : botbuilderUserMessageBackground;
-        botbuilderUserMessageTextColor = settings.userMessageTextColor
-          ? settings.userMessageTextColor
-          : botbuilderUserMessageTextColor;
-        botbuilderBotMessageBackground = settings.botMessageBoxBackground
-          ? settings.botMessageBoxBackground
-          : botbuilderBotMessageBackground;
-        botbuilderBotMessageTextColor = settings.botMessageTextColor
-          ? settings.botMessageTextColor
-          : botbuilderBotMessageTextColor;
-        botbuilderIconColor = settings.botIconColor
-          ? settings.botIconColor
-          : botbuilderIconColor;
-        botbuilderIconImg = settings.botIconImage
-          ? settings.botIconImage
-          : botbuilderIconImg;
-        applyTheme();
-      }
-    },
-    error: function(e) {
-      console.log(e);
-    },
-  });
+  script = document.createElement('script');
+  script.innerHTML = getThemeCallBack;
+
+  script.type = 'text/javascript';
+  script.src = `${api_url}/cms/getSkillMetadata.json?userid=${userid}&group=${group}&language=${language}&skill=${skill}?callback=getThemeCallBack'`;
+
+  document.querySelector('body').insertAdjacentHTML('beforeend', script);
 }
 
 // to apply custom theme
 function applyTheme() {
   // body background
-  $('.susi-sheet-content-container').css(
-    'background-color',
-    botbuilderBackgroundBody,
-  );
+
+  getComputedStyle(document.querySelector('.susi-sheet-content-container'))[
+    'background-color'
+  ] = botbuilderBackgroundBody;
+
   if (botbuilderBodyBackgroundImg) {
-    $('.susi-sheet-content-container').css(
-      'background-image',
-      "url('" + botbuilderBodyBackgroundImg + "')",
-    );
+    getComputedStyle(document.querySelector('.susi-sheet-content-container'))[
+      'background-image'
+    ] = "url('" + botbuilderBodyBackgroundImg + "')";
```

Comment:
I think we can use ES6 template literals here

Reply:
it's not necessary the soul purpose is to remove jQuery